src/OpenTelemetry/CHANGELOG.md

```diff
@@ -9,16 +9,20 @@ please check the latest changes
 
 ## Unreleased
 
-* Added `ForceFlush` to `TracerProvider`. ([#1837](https://github.com/open-telemetry/opentelemetry-dotnet/pull/1837))
+* Updated the method name of `AddLegacyActivity` to `AddLegacySource`
+  [#1860](https://github.com/open-telemetry/opentelemetry-dotnet/pull/1860)
 
-* Added a TracerProvierBuilder extension method called
-  `AddLegacyActivityOperationName` which is used by instrumentation libraries
-  that use DiagnosticSource to get activities processed without
-  ActivitySourceAdapter.
+* Added `ForceFlush` to `TracerProvider`.
+  ([#1837](https://github.com/open-telemetry/opentelemetry-dotnet/pull/1837))
+
+* Added a TracerProviderBuilder extension method called `AddLegacyActivity`
```

Comment:
this is good enough.

Reply:
Should I update the method name to `AddLegacySource` here? We would want CHANGELOG.md to show that the public API with the new name was added, right?

Reply:
Updated the CHANGELOG.md